test/unit/googleError.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import {describe, it} from 'mocha';
+import * as fs from 'fs';
+import * as path from 'path';
+import {GoogleErrorDecoder} from '../../src/googleError';
+
+describe('gRPC-google error decoding', () => {
+  it('decodes error', () => {
+    // example of an actual google.rpc.Status error message returned by Language API
+    const fixtureName = path.resolve(__dirname, '..', 'fixtures','badRequest.bin');
```

Comment:
This should work, but redundant to me. Because you need create 10 '.bin' file to get buffers for 10 messages in [error_details.proto](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto). 

Let's find a better way to store the buffers. One of possible solution is that we could store in a JSON file which have keys for error details types and value is the buffers.

@alexander-fenster do you have other suggestion? Thanks